scripts/components/table/table.component.js

```diff
@@ -1,25 +1,30 @@
 import TableTemplate from 'ejs!templates/table/table.ejs';
-// import TableTopTemplate from 'ejs!templates/table/tableTop.ejs';
 import 'whatwg-fetch';
 
 export default class {
   constructor(value) {
     this.data = null;
-    this.getData()
+    this.getData(value)
       .then(response => response.json())
       .then((json) => {
         this.data = json;
         this.render(value);
       });
   }
 
-  getData() {
-    return fetch('factsheets/table.json');
+  getData(value) {
+    if(value === 'top'){
+      return fetch('factsheets/table2.json');
+    }
+    if(value === 'municipalities'){
+      return fetch('factsheets/table.json');
+    }
   }
 
-  render(value) {
-    var areamunicipalities = document.querySelector('.js-municipalities-table');
-    const template = TableTemplate({actors: this.data});
+  render() {
+    var template;
+    var areamunicipalities = document.querySelector('.js-table');
```

Comment:
This element where the table is going to be render should be setted as a setting of the component.